src/bot.ts

```diff
@@ -37,6 +37,11 @@ export default (app: Probot) => {
       );
       return;
     }
+    // Check if organization is valid
```

Comment:
@tyler415git can you change line `27` from 

`const organization = context.payload.organization?.login ?? '';`

to `const organization = context.payload.repository.owner.login;`;